data/virtualization/autoyast/host_15.xml.ep

```diff
@@ -0,0 +1,305 @@
+    <patterns config:type="list">
+      % for my $pattern (@$patterns) {
+      <pattern><%= $pattern %></pattern>
+      % }
+    </patterns>
+  </software>
+  <ssh_import>
+    <copy_config config:type="boolean">false</copy_config>
+    <import config:type="boolean">false</import>
+  </ssh_import>
+  <timezone>
+    <hwclock>UTC</hwclock>
+    <timezone>UTC</timezone>
+  </timezone>
+  <users config:type="list">
+    <user>
+      <authorized_keys config:type="list"/>
+      <encrypted config:type="boolean">true</encrypted>
+      <fullname>bernhard</fullname>
+      <gid>100</gid>
+      <home>/home/bernhard</home>
+      <home_btrfs_subvolume config:type="boolean">false</home_btrfs_subvolume>
+      <password_settings>
+        <expire/>
+        <flag/>
+        <inact/>
+        <max>99999</max>
+        <min>0</min>
+        <warn>7</warn>
+      </password_settings>
+      <shell>/bin/bash</shell>
+      <uid>1000</uid>
+      <user_password>$6$QSJG7nUenDgf$UcRX/Q6YWM2/S55RJ8Mgvu93oKEDlOs.7/rOTMYjjxXQPiZ/rq4ogx5EpkksVRZXoEF0kX8FeEa.gOTxl1l/t0</user_password>
+      <username>bernhard</username>
+    </user>
+    <user>
+      <authorized_keys config:type="list"/>
+      <encrypted config:type="boolean">true</encrypted>
+      <fullname>root</fullname>
+      <gid>0</gid>
+      <home>/root</home>
+      <home_btrfs_subvolume config:type="boolean">false</home_btrfs_subvolume>
+      <password_settings>
+        <expire/>
+        <flag/>
+        <inact/>
+        <max/>
+        <min/>
+        <warn/>
+      </password_settings>
+      <shell>/bin/bash</shell>
+      <uid>0</uid>
+      <user_password>$6$0bUrc6YvA/qw$h1Z3pzadaxmc/KgcHRSEcYoU1ShVNymoXBaRAQZJ4ozVhTbCvdAMbGQrQAAX7cC9cLRybhsvDio3kBX/IB3xj/</user_password>
+      <username>root</username>
+    </user>
+  </users>
+  <scripts>
+    <post-scripts config:type="list">
+      <script>
+        <filename>debug_libvirtd.sh</filename>
+        <source><![CDATA[
+cat >> /etc/libvirt/libvirtd.conf<< EOF
+log_level = 1
+log_filters="1:qemu 3:remote 4:event 3:util.json 3:rpc"
+log_outputs="1:file:/var/log/libvirt/libvirtd.log"
+EOF
+cp /usr/lib/systemd/network/99-default.link  /etc/systemd/network/99-default.link
+sed -i s/MACAddressPolicy=persistent/MACAddressPolicy=none/ /etc/systemd/network/99-default.link
+
+mkdir -p /root/.config/fontconfig
+cat >> /root/.config/fontconfig/fonts.conf <<EOF
+<?xml version="1.0"?>
+<!DOCTYPE fontconfig SYSTEM "fonts.dtd">
+<fontconfig> 
+  <alias>
+    <family>sans-serif</family>
+    <prefer>
+      <family>Cantarell</family>
+    </prefer>
+  </alias>
+</fontconfig>
+EOF
+]]>
+        </source>
+      </script>
+    % if ($check_var->('VERSION', '15')) {
+      <script>
+        <filename>disable_dom0_balloon.sh</filename>
+        <source><![CDATA[
+if [ -e /etc/xen/xl.conf ]; then sed -i '/autoballoon=/ s/.*/autoballoon="off"/' /etc/xen/xl.conf; fi
+]]>
+        </source>
+      </script>
+   % } 
+    </post-scripts>
+  </scripts>
+</profile>
```

Comment:
The two files host_12 | host_15.xml.ep have a lot in common. Why not making them in one? What's the considerations?

Reply:
There is much syntax change between sle12 autoyast  and sle115.  We have to have two templates.

Reply:
ok, make sense.